benchmark/storage/data_table_benchmark.cpp

```diff
@@ -41,8 +41,8 @@ class DataTableBenchmark : public benchmark::Fixture {
 
   // Test infrastructure
   std::default_random_engine generator_;
-  storage::BlockStore block_store_{1000};
-  common::ObjectPool<storage::BufferSegment> buffer_pool_{num_inserts_};
+  storage::BlockStore block_store_{1000, 1000};
+  common::ObjectPool<storage::BufferSegment> buffer_pool_{num_inserts_, num_inserts_};
```

Comment:
let's actually have descriptive variable names here instead of reusing `num_inserts_`